### PR TITLE
Rls 0.23.0rc1

### PR DESCRIPTION
**Pull request by TomAugspurger**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,23 +1,22 @@
...
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
   url: https://github.com/pandas-dev/pandas/releases/download/v{{ version }}/{{ name }}-{{ version }}.tar.gz
   sha256: 4cdec5f000d2988cf791de590b94037fa52e198c31f07278dd889cc13cf21d6f
 
 build:
   number: 0
-  script: python setup.py install --single-version-externally-managed --record=record.txt
+  script: python -m pip install --no-deps --ignore-installed .
```
Comment (ocefpaf) [inline recipe/meta.yaml]: See https://conda-forge.org/docs/meta.html#use-pip

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,23 +1,22 @@
 {% set name = "pandas" %}
-{% set version="0.23.0rc1" %}
+{% set version = "0.23.0rc1" %}
```
Comment (ocefpaf) [inline recipe/meta.yaml]: We actually need this to the bot can parse the version and send PRs updating the package. I'll check if the `master` branch needs this change too.

Comment (TomAugspurger) [reply recipe/meta.yaml]: Thanks. If it’s easier I was just planning to merge this to master, even though it’s just an RC. Will that cause issues?

Comment (TomAugspurger) [reply recipe/meta.yaml]: Yeah, I just didn't want to mess anything up by trying to do that myself :) To get a `dev` branch, should I

1. Branch `dev off the current master
2. Cherry pick https://github.com/conda-forge/pandas-feedstock/pull/41/commits/7bd7d85edbdbd8bdcee851d3daf68bd1315e797c
3. Push that branch to upstream

Is there any way to skip the CI builds when I push upstream? Will `[ci skip]` do it?

**Comment by TomAugspurger** [comment]
Ok, I subscribed there for updates. Our build is also apparently has a hardcoded path in it somewhere that conda-build doesn't like. Trying to reproduce locally.

**Comment by ocefpaf** [comment]
> Ok, I subscribed there for updates. Our build is also apparently has a hardcoded path in it somewhere that conda-build doesn't like. Trying to reproduce locally.

Don't worry about it. We can merge with the failing status and the package will be uploaded, or I can remove the latest `conda-build` either way there nothing we need to do here in this feedstock.

BTW, I created a branch for the dev releases:

https://github.com/conda-forge/pandas-feedstock/tree/dev

do you mind pointing this PR there?

**Comment by TomAugspurger** [comment]
Thanks, I changed the base branch to `dev`.

**Comment by ocefpaf** [comment]
BTW, if you re-render with `master` version of conda-smithy it should fix the failures here. I'm working on a release...

**Comment by TomAugspurger** [comment]
OK, re-rendered w/ master.

**Comment by TomAugspurger** [comment]
> We can merge with the failing status and the package will be uploaded

@ocefpaf to clarify, do you think the failure at https://ci.appveyor.com/project/conda-forge/pandas-feedstock/build/1.0.88/job/056twhw62n7wn3f6#L4381 is OK? That seems like it's halting the build, and will have to be addressed by a new sdist from pandas.

**Comment by ocefpaf** [comment]
> @ocefpaf to clarify, do you think the failure at ci.appveyor.com/project/conda-forge/pandas-feedstock/build/1.0.88/job/056twhw62n7wn3f6#L4381 is OK? 

Nope. I meant the CircleCI failure, The ApVeyor failure is real :smile: 

The previous released is passing in #42 and has virtually the change recipe changes as this PR. Do you think it may be something with the RC release? I'll take a closer look at the logs soon.

**Comment by ocefpaf** [comment]
This hard-coded path seems odd `/Users/taugspurger/sandbox/pandas/pandas/_libs/algos.pxd` :-/

See https://ci.appveyor.com/project/conda-forge/pandas-feedstock/build/1.0.88/job/056twhw62n7wn3f6#L4381

**Comment by TomAugspurger** [comment]
Yep, definitely a problem with our build script :D

I'll tag fix that, tag and rc2, and then update the SHA here.

**Comment by TomAugspurger** [comment]
Thanks for all your help @ocefpaf!

**Comment by ocefpaf** [comment]
Glad I could help. Thanks for the RC release, looking forward to trying it out!

File: recipe/meta.yaml [hunk 3]
```diff
@@ -1,39 +1,38 @@
...
+    - pip
     - cython
     - numpy 1.9.*  # [py27 or py35]
     - numpy 1.11.*  # [py36]
   run:
     - python
     - numpy >=1.9.*  # [py27 or py35]
     - numpy >=1.11.*  # [py36]
-    - python-dateutil
+    - python-dateutil >=2.5.0
```
Comment (jreback) [inline recipe/meta.yaml]: thanks @TomAugspurger yeah meant to update this :>

**Pull request merged**